app/src/main/java/nerd/tuxmobil/fahrplan/congress/favorites/StarredListFragment.java

```diff
@@ -303,10 +302,23 @@ private void askToDeleteAllFavorites() {
 
     public void deleteAllFavorites() {
```

Comment:
Is there a difference between favorite-, starred-, highlighted-item? If not, I propose to only use a single name across code base. Maybe a different issue is required for such a refactoring.